### PR TITLE
Draft delete by form id

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA:  https://aottech.atlassian.net/browse/FWF-1742
Issue Type: BUG/ FEATURE 

# Changes
  1. when the form delete then will delete it's drafts too

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]
delete draft by formid

**Comment by abhilash-aot** [comment]
Resolve conflict

File: forms-flow-web/src/components/Form/List.js [hunk 1]
```diff
@@ -749,6 +750,10 @@ const mapDispatchToProps = (dispatch, ownProps) => {
       if (!applicationCount) {
         dispatch(deleteForm("form", formId));
       }
+      // delete drafts when form delete
+      deleteDraftbyFormId(formId).catch(err=>{
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Form/List.js:754]: Cant we trigger this from backend itself . Add code to call draft delete function when form delete is done @sreehari-aot / @shuhaib-aot

**Review by abhilash-aot — CHANGES_REQUESTED** [review]
We dont need to have seperate api  call when doing form delete. Should be handled at backend with the form delete side effect

File: forms-flow-web/src/components/Form/List.js (hunk 1 above)
Comment (sreehari-aot) [reply forms-flow-web/src/components/Form/List.js:754]: better in that way @abhilash-aot

Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/List.js:754]: ok

**Comment by shuhaib-aot** [comment]
Will open a new PR based on the conversation

**Pull request closed without merging**